### ISSUE TITLE
Questions about training progress

### ISSUE THREAD

**Issue by alala521** [KuanchihHuang/Reason3D#11]
Here are the training details of the paper I would like to inquire about:
1.For the RES task, is it trained on the training datasets of Nr3D and ScanRefer, and tested on the test dataset of ScanRefer?
2.For the Reason3D task, is it only trained on the Reason3D dataset and then tested on the test set?
3.Will the code for the 3DQA task be made public?
Thank you for your response!

**Comment by KuanchihHuang** [comment]
1. Currently trained on ScanRefer and evaluated on its validation set. If you intend to further train on Nr3D, please ensure no overlapping data between these two datasets to avoid data leakage.
2. For the release version, we pre-train on the referring segmentation task and then fine-tune on the Reason3D dataset.
3. We currently have no plans to release this component.
